deploy/runtools/runtime_config.py

```diff
@@ -264,6 +265,11 @@ def __init__(self, runtimeconfigfile, configoverridedata):
         self.m4_16xlarges_requested = int(runtime_dict['runfarm']['m4_16xlarges']) if 'm4_16xlarges' in runtime_dict['runfarm'] else 0
         self.f1_2xlarges_requested = int(runtime_dict['runfarm']['f1_2xlarges']) if 'f1_2xlarges' in runtime_dict['runfarm'] else 0
 
+        if 'launch_instances_timeout_minutes' in runtime_dict['runfarm']:
+            self.launch_timeout = timedelta(minutes=int(runtime_dict['runfarm']['launch_instances_timeout_minutes']))
+        else:
+            self.launch_timeout = timedelta(minutes=60)
```

Comment:
I default to a one-hour timeout if it isn't specified.  I suppose, some people might want the old behavior (if they care about making sure that they have minimized the number of availability zones.  Let me know if you disagree with this default.  I'm happy to change it to 0 so that the old behavior is the default.

Reply:
I would prefer to keep the original behavior if there's no launch_instances_timeout_minutes key, just so that there are no surprises for users relying on it to error if there aren't enough instances. 

IIRC some of the isca workloads that require many instances expect this behavior and we probably wouldn't want it to keep retrying for an hour while leaving the most of the instances running if someone starts the script and walks away.

For the explicit default later, I think 60 mins is fine.

Reply:
When the key isn't included in runtime.ini, the timeout defaults to 0 now.